src/creditPreview/creditPreview.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace StarkInfra;
+use StarkInfra\Utils\API;
+use StarkInfra\Utils\Rest;
+use StarkInfra\Utils\Checks;
+use StarkInfra\Utils\SubResource;
+
+
+class CreditPreview extends SubResource
+{
+    /**
+    # CreditPreview object
+
+    A CreditPreview is used to get information from a credit before taking it.
+    This resource can be used to preview credit notes
+
+    ## Parameters (required):
+        - credit [CreditNotePreview Object]: Information preview of the informed credit.
+        - type [string]: Credit type. Options: "credit-note"
+     */
+    function __construct(array $params)
+    {
+        $this-> credit = Checks::checkParam($params, "credit");
+        $this-> type = Checks::checkParam($params, "type");
+
+        Checks::checkParams($params);
+        
+        list( $this->credit, $this->type) = self::parseCredit( $this->credit, $this->type);
```

Comment:
ta com um espaço antes do $this em dois lugares aqui

Reply:
check